maps/Hydration/HDRUK/2.2.1/translation.jsonata

```diff
@@ -0,0 +1,221 @@
+(
+
+$isHidden := function($location){
+    $sum($map($hiddenFields, function($h){
+        $contains($location, $h) ? 1 : 0
+    })) >= 1 ? true : false
+};
+
+$hiddenFields := [
```

Comment:
It's obviously not ideal that these are hard coded but I couldn't think of a way around it.  They would either be hard coded here or on the front end.

Reply:
I think this is a better place for it. The FE will be calling the BE which will be grabbing this mapping file.

I guess we just need to be aware and maybe put in some safety net that if someone updates or touches this mapping file, it could potentially break the onboarding form.